bot.py

```diff
@@ -0,0 +1,80 @@
+import discord
+from discord.ext import commands
+import random
+from vars import *
+from tokens import * # Fill your own token
```

Comment:
We generally use [pydotenv](https://pypi.org/project/pydotenv/) to load our secrets from a `.env` file. Make sure to include it in `.gitignore`. Also make a `sample.env` file as an example.

Use `pydotenv` module instead of importing `tokens`.